annotated-projects/PXD000651/PXD000651.idf.tsv

```diff
@@ -0,0 +1,34 @@
+MAGE-TAB Version	1.1
+Investigation Title	CSF proteome mapping: Human cerebrospinal fluid LC-MS/MS, Part 1
+Experiment Description	3 mL CSF from a pool of 21 neurologically healthy donors was separated into a bound and a depleted fraction using a MARS Hu-14 column. This experiment represents the bound fraction. Gel separation on a gradient gel (SDS-PAGE) followed and the gel lane was cut into 37 fractions. Each fraction was in-gel trypsin digested and analyzed separately by LC-MS/MS on an Orbitrap Velos Pro mass spectrometer. Resulting data was searched using SeachGui, summarized in PeptideShaker and exported to CSF-PR.
+
+Date of Experiment	2013-12-17
+Public Release Date	2014-08-04
+
+Protocol Name	P-MTAB-Sample-PXD000651	P-MTAB-Data-PXD000651
+Protocol Type	sample collection protocol	data analysis protocol
+Protocol Description	Not available	Not available
+Protocol Parameters
+Protocol Hardware	instrument model;LTQ Orbitrap
+Protocol Software		PeptideShaker 0.23.0-beta
+Protocol Contact
+
+Person Last Name	Guldbrandsen
```

Comment:
note: here only one author is specified (so repeating authors is maybe a small, unimportant flaw?!)

Reply:
Agreed.